src/profile-logic/process-profile.js

```diff
@@ -1379,6 +1379,64 @@ function processMarkerSchema(geckoProfile: GeckoProfile): MarkerSchema[] {
   return combinedSchemas;
 }
 
+export function insertExternalMarkersIntoProfile(
+  markers: MixedObject,
```

Comment:
Is it possible to add a type for this?

also nit: I got confused for a second sometimes thinking that these are the markers in the thread while reading the code below. Maybe naming it `externalMarkers` would make it easier to read, what do you think?